[INFO -- 2018-11-15 20:11:52]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-15 20:11:52]
Initialization strings:
[INFO -- 2018-11-15 20:11:52]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-15 20:11:52]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-11-15 20:11:52]
initSignal:0
[INFO -- 2018-11-15 20:11:52]
endSignal:5
[INFO -- 2018-11-15 20:11:52]
faultModel:2
[EXCEPTION -- 2018-11-15 20:11:55]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-11-15 20:11:55]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-11-15 20:11:55]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad88b7 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7adea06 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7adea41 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7adec74 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b06ed2 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x00005555555582aa in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd850, __n=18446744073709551605) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x0000555555558043 in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18446744073709551605) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x0000555555557b1a in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd850, __n=18446744073709551605) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x000055555555787f in std::_Vector_base<double, std::allocator<double> >::_M_create_storage (this=0x7fffffffd850, __n=18446744073709551605) at /usr/include/c++/7/bits/stl_vector.h:187
#11 0x0000555555556f4f in std::_Vector_base<double, std::allocator<double> >::_Vector_base (this=0x7fffffffd850, __n=18446744073709551605, __a=...) at /usr/include/c++/7/bits/stl_vector.h:138
#12 0x00005555555569fb in std::vector<double, std::allocator<double> >::vector (this=0x7fffffffd850, __x=std::vector of length -10, capacity -10 = {...}) at /usr/include/c++/7/bits/stl_vector.h:328
#13 0x0000555555555f5f in train (iter1=45000, iter2=45000, N=4, alpha1=2.2222199999999999e-05, alpha2=2.2222199999999999e-05, stopsWhenStable=true) at lreg.cpp:199
#14 0x0000555555556467 in main (argc=3, argv=0x7fffffffde68) at lreg.cpp:253

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Memory content before bitflip:0x020x000x000x00
Fault Model: Random bit-flip
base address to flip value: 0x7fffffffd900
address max offset: 4
Memory content after  bitflip:0x6e0xfd0xb20xc3
frame name: main
symbol name: i2
symbol filename: lreg.cpp
symbol line: 232
value: 3283287406
value address: 0x7fffffffd900
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2018-11-15 20:11:55]
Fault Injection Successful after 3.705500602722168s
[INFO -- 2018-11-15 20:11:55]
event type: exit
[EXCEPTION -- 2018-11-15 20:11:55]
exit code: no exit code available
